tests/State/ScopeTest.php

```diff
@@ -131,7 +122,7 @@ public function testAddEventProcessor(): void
             return null;
         });
 
-        $scope->addEventProcessor(function () use (&$callback3Called): ?Event {
+        $scope->addEventProcessor(function () use (&$callback3Called) {
```

Comment:
why did you remove ?Event return value?

Reply:
I said to you in another PR to add them but rethinking about it it makes no sense if the only return type of the lambda is an hardcoded `null` value